### ISSUE TITLE
new Git bash path

### ISSUE THREAD

**Issue by cottsak** [MRCollective/TortoiseGitToolbar#10]
I needed to do this on the command line to get the `CTRL+G, B` to work because [this path is wrong](https://github.com/MRCollective/TortoiseGitToolbar/blob/master/TortoiseGitToolbar/Config/Constants/PathConfiguration.cs#L10):

```
mklink /d "C:\Program Files (x86)\git" "C:\Program Files\Git
```

**Comment by MattDavies** [comment]
The launcher needs to be changed to check a number of possible standard paths for the executables. In the future ideally we'd allow configuration via Visual Studio.

**Comment by YueLinHo** [comment]
For a ref.

TortoiseGit's setting:
![image](https://cloud.githubusercontent.com/assets/5831804/12776062/e04244f4-ca8d-11e5-9a88-19836c061ef2.png)

It's saved in `HKEY_CURRENT_USER\Software\TortoiseGit\MSysGit`

**Comment by cottsak** [comment]
@MattDavies Looks like https://github.com/MRCollective/TortoiseGitToolbar/pull/9 will fix this immediate problem for now. Can we merge it?

**Comment by MattDavies** [comment]
Fixed by #13, will be released as 1.0.
